### DISCUSSION TITLE
Build stuck in queue

### DISCUSSION THREAD

**Discussion by Mat40** [codemagic-ci-cd/codemagic-docs#2219] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Configure workflow
Start new build

### Expected results

Build Start

### Actual results

Build stuck in queue

### Build id (optional)

6561c73a2b8a5431ff0cd258

**Comment by mikhail-tokarev** [comment, marked as answer]
hi!
there was longer queue time because high activity from crypto miners who used significant amount of macOS machines. It took some time to filter them out and ban but everything is ok now. Sorry for the inconvenience!
